Kernel/KerrGeodesics.m

```diff
@@ -1,10 +1,16 @@
 (* ::Package:: *)
 
+$BaseDirectory
```

Comment:
Is this supposed to be here?

Reply:
No its removed now, also seems as though a performed the pull request on a version which was a few days out of date in which the test files weren't completed yet, the correct form of the wlt should now be included.